src/main/g8/src/main/scala/$package$/Boot.scala

```diff
@@ -55,15 +55,28 @@ object Boot extends App {
 
   def run(args: List[String]): ZIO[zio.ZEnv, Nothing, Int] =
     loadConfig.flatMap { rawConfig =>
-      val configLayer = TypesafeConfig.fromHocon(rawConfig, AppConfig.description)
+      val configLayer = ZLayer.fromEffect {
+        ZIO.fromEither {
+          TypeSafeConfigSource
```

Comment:
Should look much simpler if we use `TypesafeConfig.fromTypesafeConfig` like this:
```scala
TypesafeConfig.fromTypesafeConfig(rawConfig, AppConfig.descriptor)
```

Or am I missing something here?

Reply:
This function no longer exists in `zio-config` `1.0.0-RC16-1`

Reply:
I'm pretty sure it is :thinking: 

https://github.com/zio/zio-config/blob/master/typesafe/src/main/scala/zio/config/typesafe/TypesafeConfig.scala#L27

Reply:
Ah sorry, I just found it. Let me fix that.

Reply:
@vpavkin fixed. I didn't find this object due to the typo:
`TypesafeConfig` vs `TypeSafeConfigSource` :)

Reply:
I'll let `zio-config` devs know about that.